content/4_Gold/Combinatorics.mdx

```diff
@@ -0,0 +1,270 @@
+---
+id: combo
+title: 'Combinatorics'
+author: Jesse Choe
+prerequisites:
+  - divisibility
+  - modular
+description: Counting solutions using math.
```

Comment:
feel like including "math" in this description doesn't really add any information but ok